src/service/user.service.js

```diff
@@ -0,0 +1,65 @@
+import { ApiError } from '../exeptions/api.error.js';
```

Comment:
There is a typo in the import path: 'exeptions' should be 'exceptions'. Please correct the path to '../exceptions/api.error.js'.